UI/Tests/BaseTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+using NUnit.Framework;
+using System;
+using UI.WebDriver;
+using OpenQA.Selenium;
+
+namespace UI.Tests
+{
+	public class BaseTest
+	{
+		protected static IWebDriver Driver;
+
+
+		[SetUp]
+		public void Setup()
+		{
+			Driver = Browser.GetDriver();
+			Browser.WindowMaximaze();
+			Browser.StartNavigate();
+		}
+
+		[TearDown]
+		public void Quite()
+		{
+			Browser.QuiteBrowser();
+		}
+
+		[OneTimeTearDown]
```

Comment:
Delete that empty method. If method does nothing, then we do not need it. It is common code clean approach